### ISSUE TITLE
Allow to change Axis Type

### ISSUE THREAD

**Issue by RASK18** [EPPlusSoftware/EPPlus#267]
Hello! Could you allow to change the AxisType somehow?
I found this function in the code but it is not public :(

https://github.com/EPPlusSoftware/EPPlus/blob/96a7ac124515c41ce6336a64ed377549300d440a/src/EPPlus/Drawing/Chart/ExcelChartAxis.cs#L510

**Comment by JanKallman** [comment]
No, sorry, there is no way of doing this at the moment, but we should look into it. I'll add this as an enhancment.
We will have to add a lot of functionality and validations to the different chart types to switch this method to public.

**Comment by OssianEPPlus** [comment]
Added in Epplus 8.1
